### ISSUE TITLE
Dialog Size assumes short WS name 

### ISSUE THREAD

**Issue by DanielMurphy22** [mantidproject/mantid#28827]
**Original reporter:** Stephen King @smk78 
The Algorithm Dialog box currently assumes that Workspace names are less than a certain length and does not resize horizontally for particularly long ws names. We should not make this assumption.
### Expected behavior
The algorithm Dialog appears large enough to show all functionality 
### Actual behavior
In this case, the user had a rather long ws name, so the "Replace InputWorkspace name as Output WS name" button was not visible, as the Dialog did not horizontally resize. 
### Steps to reproduce the behavior
run `SupercalifragilisticexpialidociousSomethingQuiteAtrocious = CreateSampleWorkspace()`
Right-click on the workspace and select Rename. Can you see the 
<img width="28" alt="Screenshot 2020-06-11 at 09 27 54" src="https://user-images.githubusercontent.com/55980573/84375670-d64e5e80-abd7-11ea-996e-87f8cc93cadc.png"> button? 
This issue may manifest itself differently for different screens / OSs.
Basically, the extent of the InputWorkspace box is going beyond the dialog edge.
### Platforms affected
Plot and WB all, noticed in Plot 5.0 on Windows

**Comment by martyngigg** [comment]
@clayton-tom Did you make any headway with this? I think it ended up being way more complicated than expected?

**Comment by smk78** [comment]
If that is the case, can you do anything that makes it more obvious that more functionality is available if you stretch the dialog?

**Comment by clayton-tom** [comment]
I wasn't able to get it done before the end of the sprint, it turned out to be an issue in the code for the generic Algorithm dialogue creation but I couldn't find out where/why at the time. I've still got some notes on it, so happy to look into it again at some stage

**Comment by martyngigg** [comment]
@clayton-tom Thanks. Could you put the notes here so that someone else could pick it up?

**Comment by clayton-tom** [comment]
![AlgorithmDialog](https://user-images.githubusercontent.com/68227228/95309915-ab250480-0883-11eb-8620-c7f52e0cfb29.png)
